i18n/en.i18n.json

```diff
@@ -64,7 +64,7 @@
   "activity-unchecked-item": "unchecked %s in checklist %s of %s",
   "activity-checklist-added": "added checklist to %s",
   "activity-checklist-removed": "removed a checklist from %s",
-  "activity-checklist-completed": "completed checklist __checklist__ at card __card__ at list __list__ at swimlane __swimlane__ at board __board__",
+  "activity-checklist-completed": "completed checklist %s of %s",
```

Comment:
Please change bottom of this file to:
```
  "newUserPopup-title": "New User",
  "notifications": "Notifications",
  "view-all": "View All",
  "filter-by-unread": "Filter by Unread",
  "mark-all-as-read": "Mark all as read"
```

Reply:
I did add these translation strings already to Wekan, so that they will be translated.